includes/languages/english.php

```diff
@@ -570,7 +570,7 @@ function zen_date_raw($date, $reverse = false) {
 
   define('DB_ERROR_NOT_CONNECTED', 'Error - Could not connect to Database');
   define('ERROR_DATABASE_MAINTENANCE_NEEDED', '<a href="https://docs.zen-cart.com/user/troubleshooting/error_71_maintenance_required/" rel="noopener" target="_blank">ERROR 0071 There appears to be a problem with the database. Maintenance is required.</a>');
-
+  define('ERROR_PAGE_NOT_FOUND', 'Sorry, the page you were attempting to access cannot be found.');
```

Comment:
Is this additional language define necessary? 

And doesn't it duplicate the one in page.php which will therefore cause an Already Defined notice?

Reply:
Yes, you're right - it's a relic of the first approach.